Deep_Learning/HoneypotAI/ml/attack_classifier.py

```
@@ -0,0 +1,270 @@
+"""
+Attack Classifier for HoneypotAI
+Uses supervised learning to classify known attack types
+"""
+
+import numpy as np
+import pandas as pd
+from typing import Dict, List, Any, Optional, Tuple
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import: `typing.Optional`

(F401)